src/components/scroll-to.cjsx

```diff
@@ -0,0 +1,65 @@
+React = require 'react'
+_ = require 'underscore'
+
+# Note that the Position mixin methods are called directly rather than mixing it in
+# since we're a mixin ourselves our consumers also include GetPosition and it causes
+# duplicate method name errors to mix it in
+Position = require './get-position-mixin'
+
+DEFAULT_DURATION   = 750 # milliseconds
+DEFAULT_TOP_OFFSET = 80  # pixels
```

Comment:
I'd like it to be, but not sure how to share var's between CSS and JS.  Is there a pattern for that?  The navbar is 69px in total, so 80 gives it just a bit more space at the top to look nicer.